deploy/helm_charts/envs/prod.yaml

```diff
@@ -42,13 +42,15 @@ serviceAccount:
 nl:
   enabled: true
   embeddingsSpec:
-    defaultIndex: "medium_ft"
-    enabledIndexes: [
-      "bio_ft"
-      "medium_ft",
-      "sdg_ft",
-      "undata_ft",
-    ]
+    defaultIndex: "base_uae_mem"
+    enabledIndexes:
+      ["base_uae_mem", "bio_ft", "medium_ft", "sdg_ft", "undata_ft"]
```

Comment:
need bio_ft, since its not launched?

Reply:
Removed bio_ft in all non bio instances now.